libs/Zend/Session/Exception.php

```diff
@@ -55,10 +56,12 @@ class Zend_Session_Exception extends Zend_Exception
      */
     static public function handleSessionStartError($errno, $errstr, $errfile, $errline, $errcontext)
     {
-        if (!isset(self::$sessionStartError)) {
-            self::$sessionStartError = '';
+        $message = $errfile . '(Line:' . $errline . '): Error #' . $errno . ' ' . $errstr;
+        if (E_ERROR === $errno || E_CORE_ERROR === $errno || E_COMPILE_ERROR === $errno) {
```

Comment:
only trigger a session start exception when one of these 3 errors occur. Those 3 should be enough and no need to handle recoverable error etc.